src/vcr_clj/cassettes/serialization.clj

```diff
@@ -126,12 +126,13 @@
    :map-delimiter      ",",
    :print-color        false,
    :print-fallback     :print,
-   :print-handlers     print-handlers
    :print-meta         false,
    :sort-keys          true,
    :strict             false,
    :width              80})
 
 (defn print
-  [cassette]
-  (printer/pprint cassette puget-options))
+  [cassette & [{:keys [print-handlers]
+                :or {print-handlers default-print-handlers}}]]
```

Comment:
doesn't the use of `or` here imply that users interested in simply adding functionality (e.g., serializing a custom type that appears as an argument) will need to manually combine their print-handlers with `default-print-handlers`? I'm wondering if it would be better to combine with `some-fn`; I think that would be more in line with the `merge` you're doing in the data-reader case, and my initial guess is that it would more closely match most use cases